### ISSUE TITLE
Profile: Image Upload Broken

### ISSUE THREAD

**Issue by Partyschaum** [Jimdo/reportbook#432]
I just tried to upload a profile image to my "Ausbilderprofil" and got the following error message:
```

Fatal error: Uncaught TypeError: Return value of Jimdo\Reports\Profile\Profile::trainingYear() must be of the type integer, string returned in /var/www/src/Profile/Profile.php:129
Stack trace:
#0 /var/www/src/Serializer.php(93): Jimdo\Reports\Profile\Profile->trainingYear()
#1 /var/www/src/Profile/ProfileMongoRepository.php(59): Jimdo\Reports\Serializer->serializeProfile(Object(Jimdo\Reports\Profile\Profile))
#2 /var/www/src/Profile/ProfileService.php(170): Jimdo\Reports\Profile\ProfileMongoRepository->save(Object(Jimdo\Reports\Profile\Profile))
#3 /var/www/src/Application/ApplicationService.php(776): Jimdo\Reports\Profile\ProfileService->editImage('58aac1245c25a', 'iVBORw0KGgoAAAA...', 'png')
#4 /var/www/src/Web/Controller/UserController.php(123): Jimdo\Reports\Application\ApplicationService->editImage('58aac1245c25a', 'iVBORw0KGgoAAAA...', 'png')
#5 /var/www/src/Web/Router.php(117): Jimdo\Reports\Web\Controller\UserController->uploadAction()
#6 /var/www/app/router.php(15): Jimdo\Reports\Web\Router->dispatch('/user/uploa in /var/www/src/Profile/Profile.php on line 129
```

**Comment by Partyschaum** [comment]
I succeeded in uploading another image (`.jpg`) but the upload seems not to work properly nevertheless. Uploading (`.jpeg`) does nothing. Repeated uploading of different images behaves 'flaky'.

One time I got:
```
Warning: file_get_contents(): Filename cannot be empty in /var/www/src/Web/Controller/UserController.php on line 121

Warning: Cannot modify header information - headers already sent by (output started at /var/www/src/Web/Controller/UserController.php:121) in /var/www/src/Web/Controller/Controller.php on line 108
```

**Comment by tomstich** [comment]
Implemented in #435
